src/app/shared/ui-components/error-message/error-message.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Is this file autogenerated or you created it ?

Reply:
auto-generated